src/txHashBuilder.h

```diff
@@ -91,9 +115,27 @@ typedef struct {
 	};
 
 	tx_hash_builder_state_t state;
+	tx_hash_builder_output_substate_t subState;
 	blake2b_256_context_t txHash;
 } tx_hash_builder_t;
 
+typedef enum  {
+	LEGACY = 0,
+	POST_ALONZO = 1
```

Comment:
As agreed elsewhere, we prefer to use names that are more perpetual (won't become invalid after the next serialization format change):
ARRAY_LEGACY and MAP_BABBAGE. You can add the CDDL names as comments to the enum values.